### PR TITLE
LLP-132- Add Fact Sheet Component with redux and firebase

### PR DESCRIPTION
**Pull request by mdainys**

### CODE REVIEW

File: src/components/FactSheetsData.js [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+
+export const FactSheetsData = () => {
+  const { t } = useTranslation("common");
+  const router = useRouter();
+
+  const factsheetsData = useSelector(getFactsheetsState);
+
+  const [factSheetList, setFactSheetList] = useState([]);
+  const [activeFactSheetList, setActiveFactSheetList] = useState([]);
+  function factsheetTabHandler(e) {
```
Comment (pinenetcoder) [inline src/components/FactSheetsData.js:17]: Add a space and do arrow function please :)

File: src/pages/financial-reporting.js [hunk 2]
```diff
@@ -81,7 +60,7 @@ export default function FinancialReporting() {
             " "
           )}
         >
-          <div className={styles.ReportsWrapper}>
+          {/* <div className={styles.ReportsWrapper}>
```
Comment (pinenetcoder) [inline src/pages/financial-reporting.js:63]: Remove commented Area, please :)

Comment (mdainys) [reply src/pages/financial-reporting.js:63]: done

File: src/components/FactSheetsData.js (hunk 1 above)
Comment (mdainys) [reply src/components/FactSheetsData.js:17]: done

**Review by pinenetcoder — APPROVED** [review]

**Pull request merged**